### PR TITLE
4290 - Improve collection page edit

### PR DESCRIPTION
**Pull request by WillNigel23**

### CODE REVIEW

**Comment by saracarl** [comment]
looks fine, other than an HTR nav to nowhere!  (and failing tests...)

**Comment by WillNigel23** [comment]
Still WIP. Please do not merge to dev yet.

**Comment by benwbrum** [comment]
I've merged this into 3890.  Should I delete the branch and PR or are you still working on it, @WillNigel23 ?

File: app/models/page.rb [hunk 1]
```diff
@@ -89,6 +89,14 @@ class Page < ApplicationRecord
 
   serialize :metadata, Hash
 
+  ACCEPTED_FILE_TYPES = [
+    'image/jpeg',
+    'image/png',
+    'image/gif',
+    'image/bmp',
+    'image/tiff'
+  ].freeze
```
Comment (WillNigel23) [inline app/models/page.rb:98]: Please review if I have covered all accepted files for page images here

Comment (saracarl) [reply app/models/page.rb:98]: [Documentation says](https://content.fromthepage.com/project-owner-documentation/image-upload-file-guidelines/):  Acceptable file formats: PDF, PNG, GIF, JPG, TIF

We're not supporting PDF for an individual page upload, but we should support the others.  Are we transforming them?  I think we do transform some if they are uploaded in a zip file for an initial upload.

Comment (WillNigel23) [reply app/models/page.rb:98]: Currently our filepicker component only accepts image files

Comment (WillNigel23) [reply app/models/page.rb:98]: In any case, #3890 can add more supported files here

**Comment by benwbrum** [comment]
Blocked -- waiting on HTR

**Comment by saracarl** [comment]
I'm closing this without merging -- changes are in #4423 .

**Pull request closed without merging**